src/modules/tools/endstops/Endstops.cpp

```diff
@@ -1200,6 +1200,54 @@ void Endstops::on_gcode_received(void *argument)
                 gcode->stream->printf(" will take effect next home\n");
                 break;
 
+            case 365: //M365, change endstop configs for pipette type
+                float value;
+                char pos;
+                switch(gcode->subcode) {
+                  case 0:
+                    // Homing Position
+                    for (auto &p : homing_axis) {
+                      if(gcode->has_letter(p.axis)) {
+                        pos = p.axis;
+                        value = gcode->get_value(p.axis);
```

Comment:
We should fail if this gives us `0` which looks like what it does when no value is found